VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/quick-links/quick-links.component.html

```diff
@@ -0,0 +1,68 @@
+
+<form [formGroup]="quickLinkForm" class="govuk-form-group" (ngSubmit)="onSubmit()">
+  <div [ngClass]="{ 'govuk-form-group--error': error.nameError }" class="govuk-!-margin-bottom-5">
+    <p *ngIf="error.nameError" class="govuk-error-message">{{ 'quick-participant-landing-page.name-error' | translate }}</p>
+    <label class="govuk-label" for="full-name">
+      {{ 'quick-participant-landing-page.name-label' | translate }}
+    </label>
+    <input formControlName="name" class="govuk-input" id="full-name" name="full-name" type="text" spellcheck="false" autocomplete="name">
+  </div>
+
+  <div [ngClass]="{ 'govuk-form-group--error': error.roleError }">
+    <p *ngIf="error.roleError" class="govuk-error-message">{{ 'quick-participant-landing-page.role-error' | translate }}</p>
+    <p>{{ 'quick-participant-landing-page.role-label' | translate }}</p>
+
+    <div *ngFor="let quickLinkParticipantRole of quickLinkParticipantRoles" class="govuk-radios__item">
+      <ng-container [ngSwitch]="quickLinkParticipantRole">
+        <ng-container *ngSwitchCase="role.QuickLinkParticipant">
+          <input
+            [id]="quickLinkParticipantRole"
+            formControlName="quickLinkParticipantRole"
+            class="govuk-radios__input"
+            name="quickLinkParticipantRole"
+            type="radio"
+            [value]="quickLinkParticipantRole"
+            [attr.aria-describedby]="quickLinkParticipantRole + '-item-hint'">
+          <label class="govuk-label govuk-radios__label" [attr.for]="quickLinkParticipantRole">
+            {{ 'quick-participant-landing-page.participant.role' | translate }}
+          </label>
+          <div [id]="quickLinkParticipantRole + '-item-hint'"  class="govuk-hint govuk-radios__hint">
+            {{ 'quick-participant-landing-page.participant.description' | translate }}
+          </div>
+        </ng-container>
+        <ng-container *ngSwitchCase="role.QuickLinkObserver">
+          <input
+            [id]="quickLinkParticipantRole"
+            formControlName="quickLinkParticipantRole"
+            class="govuk-radios__input"
+            name="quickLinkParticipantRole"
+            type="radio"
+            [value]="quickLinkParticipantRole"
+            [attr.aria-describedby]="quickLinkParticipantRole + '-item-hint'">
+          <label class="govuk-label govuk-radios__label" [attr.for]="quickLinkParticipantRole">
+            {{ 'quick-participant-landing-page.observer.role' | translate }}
+          </label>
+          <div [id]="quickLinkParticipantRole + '-item-hint'"  class="govuk-hint govuk-radios__hint">
+            {{ 'quick-participant-landing-page.observer.description' | translate }}
+          </div>
+        </ng-container>
+      </ng-container>
+    </div>
+  </div>
+
+  <button
+    id="continue-btn"
+    class="govuk-button govuk-!-margin-top-6"
+    data-module="govuk-button"
+    type="button"
+    (click)="onSubmit()"
```

Comment:
We already state that the form should call onSubmit when submitted on line 5. Remove the click event here and change the type to 
```suggestion
    type="submit"
```

Reply:
It would also be good to have a test that checks that the form is submitted when the button is clicked.

Reply:
Removed